tensorboard/webapp/routes/dashboard_deeplink_provider.ts

```diff
@@ -86,6 +89,12 @@ export class DashboardDeepLinkProvider extends DeepLinkProvider {
         const queryParams = experimentalPlugins.map((pluginId) => {
           return {key: EXPERIMENTAL_PLUGIN_QUERY_PARAM_KEY, value: pluginId};
         });
+        if (typeof overriddenFeatureFlags.enabledColorGroup === 'boolean') {
```

Comment:
What if the type is not boolean?
As the test below shown, we want to have "enabledColorGroup=foo" be true? Which part handle this logic?

Reply:
> we want to have "enabledColorGroup=foo" be true? Which part handle this logic?

Yes, that's my intention. When user specifies a value that is not a stringified JavaScript boolean, I think it is safe to treat it as no-value and enable the feature. Since it is easily reversible action (on user's part), I don't think this is a bad call. Do note that we have been doing this for other flags already.

Reply:
so... if it's `enabledColorGroup=foo` it will not do queryParams key-value setup and then `if (params.has(ENABLE_COLOR_GROUP_QUERY_PARAM_KEY))` will be false
how do we check the `!== 'false'` condition to still enable the `enabledColorGroup` feature?

Reply:
Resolved offline by explaining how this change integrates with FeatureFlagDataSource -> store -> Router (DashboardDeepLinkProvider)